static_src/test/server/server.js

```diff
@@ -1,6 +1,62 @@
 
+// This testing server acts as a "sidecar" for a main process given as command
+// line arguments, or simply starts the testing server.
+//
+// Sidecar pattern comes from Netflix's Prana[1] but instead of providing
+// platform services, we just provide access to the testing server.
+//
+// TODO the testing server does not require any cleanup nothing is done to stop
```

Comment:
Most servers have some kind of [stop/close](https://hapijs.com/api#server-events) event so that you can clean up any resources like database connections. In this case, we would call `server.stop()` to trigger the shutdown process. Since nothing is listening to the stop event anyway, it's not a big deal.

I'll update the TODO to be more specific: It should listen to SIGTERM/SIGINT events and call `server.stop()`.